src/app/products/products.component.ts

```diff
@@ -1,15 +1,57 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
+
+import { Product } from './product.model';
+import { ProductsService } from '../services/products.service';
 
 @Component({
   selector: 'app-products',
   templateUrl: './products.component.html',
-  styleUrls: ['./products.component.scss']
+  styleUrls: ['./products.component.css']
```

Comment:
any reason on changing this to .css?

Reply:
Oh, previously I tried with css. I forgot it. BTW Wesly's files will override it.